test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
```

Comment:
This could be a simple comment with `#` as opposed to a multi-line string. Could you please make it a comment?

Reply:
I can